hub/demo/src/app/competitions/0.5b/page.tsx

```diff
@@ -17,9 +16,21 @@ From your machine you will use \`nearai submit\` to submit your runs. From withi
 
 To participate in the competition, apply [here](https://docs.google.com/forms/d/e/1FAIpQLScInS4mHyZb_kSkD0-CMPPagyhpBKdutbAyS6YNbHJc9ZgaUw/viewform).
 `;
+
+const title = '0.5B Model Training Competition';
+
+export const metadata: Metadata = {
+  title,
+};
+
+export default function CompetitionPage() {
   return (
-    <CompetitionPage title="0.5B Model Training Competition - January 2025">
+    <Competition
+      competitionId="competition_0.5b"
+      title={title}
+      schedule="Dec 10th - Jan 15th, 2025 @ 11:59 PM UTC"
```

Comment:
We should probably update dates.

Reply:
These are the dates @SkidanovAlex recently gave me. Do we need to update them again?

Reply:
Probably we need to update them again.